From: Matthias Clasen <mclasen@redhat.com>
Date: Sun, 11 Apr 2021 23:22:46 -0400
Subject: fontchooser: Fix some since annotations

The "level", "font-features" and "language" were annotated
with the wrong version.

Bug: https://gitlab.gnome.org/GNOME/gtk/-/issues/1830
Origin: upstream, 3.24.29, commit:3fb5890e695243916773b7b7b0891caf25958fb3
---
 gtk/gtkfontchooser.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/gtk/gtkfontchooser.c b/gtk/gtkfontchooser.c
index 16df10b..576bc2e 100644
--- a/gtk/gtkfontchooser.c
+++ b/gtk/gtkfontchooser.c
@@ -110,7 +110,7 @@ gtk_font_chooser_default_init (GtkFontChooserInterface *iface)
    *
    * The level of granularity to offer for selecting fonts.
    *
-   * Since: 3.22.30
+   * Since: 3.24.1
    */
   g_object_interface_install_property
      (iface,
@@ -129,7 +129,7 @@ gtk_font_chooser_default_init (GtkFontChooserInterface *iface)
    * The selected font features, in a format that is compatible with
    * CSS and with Pango attributes.
    *
-   * Since: 3.22.30
+   * Since: 3.24.1
    */
   g_object_interface_install_property
      (iface,
@@ -146,7 +146,7 @@ gtk_font_chooser_default_init (GtkFontChooserInterface *iface)
    * selected, in a format that is compatible with CSS and with Pango
    * attributes.
    *
-   * Since: 3.22.30
+   * Since: 3.24.1
    */
   g_object_interface_install_property
      (iface,
